nano/node/rpc.cpp

```diff
@@ -3314,6 +3314,44 @@ void nano::rpc_handler::unchecked_keys ()
 	response_errors ();
 }
 
+void nano::rpc_handler::unopened ()
+{
+	std::unordered_map<nano::account, nano::uint128_t> unopened;
+	auto transaction (node.store.tx_begin_read ());
+	auto iterator (node.store.pending_begin (transaction));
+	auto end (node.store.pending_end ());
+	while (iterator != end)
+	{
+		nano::pending_key key (iterator->first);
+		nano::account account (key.account);
+		nano::pending_info info (iterator->second);
+		if (node.store.account_exists (transaction, account))
+		{
+			// Skip existing accounts
+			iterator = node.store.pending_begin (transaction, nano::pending_key (account.number () + 1, 0));
```

Comment:
Guess we need extra check for possible overflow like
iterator = (account.number () == std::numeric_limits<nano::uint128_t>::max ()) ? end : node.store.pending_begin (transaction, nano::pending_key (account.number () + 1, 0));

Reply:
uint256_t::max () of course..

Reply:
Weirdly not working due to a deleted function.

```
‘nano::store_iterator<nano::pending_key, nano::pending_info>::store_iterator(const nano::store_iterator<nano::pending_key, nano::pending_info>&)’ is implicitly declared as deleted because ‘nano::store_iterator<nano::pending_key, nano::pending_info>’ declares a move constructor or move assignment operator 
```

Reply:
I can just do normal if

Reply:
Yeah, I didn't test this draft. Nornal if or break of while is fine

Reply:
Problem was not being able to copy the end, needed to get another pending_end ()